msfr/steady/msfr_ns.i

```diff
@@ -188,24 +214,38 @@ beta6 = 0.000184087
     pressure = pressure
     u = v_x
     v = v_y
-    mu = 'mu_t'
+    mu = 'mu'
     rho = ${rho}
     block = 'fuel pump hx'
   []
-  [u_turb_viscosity]
+  [u_turbulent_diffusion_rans]
+    type = INSFVMixingLengthReynoldsStress
+    variable = v_x
+    rho = ${rho}
+    mixing_length = mixing_len
+    momentum_component = 'x'
+    u = v_x
+    v = v_y
+  []
+  [u_molecular_diffusion]
     type = FVDiffusion
     variable = v_x
-    coeff = 'mu_t'
+    coeff = 'mu'
     block = 'fuel pump hx'
   []
   [u_pressure]
     type = INSFVMomentumPressure
     variable = v_x
     momentum_component = 'x'
-    pressure = pressure
+    p = pressure
```

Comment:
don't revert to desecrated syntax

Reply:
Instead of deprecate, we should desecrate things